docs/index.yaml

```diff
@@ -1,9 +1,29 @@
 apiVersion: v1
 entries:
   keda:
+  - apiVersion: v1
+    appVersion: 1.5.0
+    created: "2020-07-08T10:57:25.2069793+02:00"
+    description: Event-based autoscaler for workloads on Kubernetes
+    digest: 07b15ddae4f1c40747de063100a668ac15d504cd0548eac0e1a04381fcaa3b37
+    home: https://github.com/kedacore/keda
+    icon: https://raw.githubusercontent.com/kedacore/keda/master/images/keda-logo-500x500-white.png
+    maintainers:
+    - email: ahmels@microsoft.com
+      name: Ahmed ElSayed
+    - email: kerkhove.tom@gmail.com
+      name: Tom Kerkhove
+    - email: zroubali@redhat.com
+      name: Zbynek Roubalik
+    name: keda
+    sources:
+    - https://github.com/kedacore/keda
+    urls:
+    - https://kedacore.github.io/charts/keda-1.5.0.tgz
+    version: 1.5.0
   - apiVersion: v1
     appVersion: 1.4.1
-    created: "2020-06-11T08:05:23.8025036+02:00"
+    created: "2020-07-08T10:57:25.2039794+02:00"
```

Comment:
Pardon my ingorance and lack of HELM knowledge, but is there a way how to not update timestamps for old releases? Or is this something that's ususal in HELM?

Reply:
This is due to `helm repo index docs` but will check if it has an option to leave it be

Reply:
https://github.com/helm/helm/issues/7363

Reply:
There is no option to skip that unfortunately - https://helm.sh/docs/helm/helm_repo_index/

Reply:
ok, thanks!